web/src/user_status.ts

```diff
@@ -46,7 +46,7 @@ const user_status_emoji_info = new Map<number, UserStatusEmojiInfo>();
 export function server_update_status(opts: {
     status_text: string;
     emoji_name: string;
-    emoji_code: string | number;
+    emoji_code: string;
```

Comment:
It looks like this just undoes previous commit; better to just remove the buggy commit than make reviewers look at incorrect code.

Check out the [Zulip commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for more details.